l10n_it_website_portal_fatturapa_sale/views/payment_templates.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  Copyright 2019 Simone Rubino - Agile Business Group
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+  -->
+
+<odoo>
+    <template id="payment_tokens_list"
+              name="Add e-invoice subjected in Payment Tokens list"
+              inherit_id="payment.payment_tokens_list">
+        <xpath expr="//div[hasclass('card')]" position="after">
+            <label>
+                <input type="checkbox"
+                       name="electronic_invoice_subjected"
+                       t-att-checked="'checked' if request.env['res.partner'].browse(partner_id).electronic_invoice_subjected else None"/>
+                I want to receive electronic invoice
```

Comment:
Per moduli simili
https://github.com/OCA/l10n-italy/blob/d5e4f1eea2540ccf1be7c5699fdbcab76a722689/l10n_it_website_portal_fatturapa/views/l10n_it_website_portal_fatturapa_templates.xml#L47
avevamo usato la forma `I request to receive electronic invoices` quindi userei la stessa
Per il resto OK

Reply:
@eLBati l'avevo preso da https://github.com/OCA/l10n-italy/blob/a772bdd953419013caa93e262ad6af0dbded597b/l10n_it_website_sale_fatturapa/views/templates.xml#L40, lo cambio lo stesso?

Reply:
La versione corretta è con `request`. Sarebbe da correggere anche `l10n_it_website_sale_fatturapa`